eng/versioning.targets

```diff
@@ -36,11 +36,20 @@
       <CrossPlatformAndHasNoBrowserTarget>true</CrossPlatformAndHasNoBrowserTarget>
   </PropertyGroup>
   
-  <!-- Enables browser warnings for cross platform or Brwoser targeted builds -->
+  <!-- Enables browser warnings for cross platform or Browser targeted builds -->
   <ItemGroup Condition="('$(TargetsBrowser)' == 'true' or '$(CrossPlatformAndHasNoBrowserTarget)' == 'true') and '$(IsTestProject)' != 'true'">
     <SupportedPlatform Include="browser"/>
   </ItemGroup>
 
+  <!-- Enables warnings for iOS targeted builds -->
+  <ItemGroup Condition="'$(TargetsiOS)' == 'true' and '$(IsTestProject)' != 'true'">
+    <SupportedPlatform Include="ios"/>
+  </ItemGroup>
+  <!-- Enables warnings for tvOS targeted builds -->
+  <ItemGroup Condition="'$(TargetstvOS)' == 'true' and '$(IsTestProject)' != 'true'">
+    <SupportedPlatform Include="tvos"/>
+  </ItemGroup>
+  
```

Comment:
Why is this needed ?

/cc @buyaa-n

Reply:
@buyaa-n Thanks, what about `tvOS`? Does it still need to be enabled here?

Reply:
Yes, for unsupported APIs the analyzer shows warnings only for the platforms included in the [MSBuild SupportedPlatform list](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedPlatforms.props#L16-L20), `tvOS` is not there so to see unsupported warning for `tvOS` you can add it to the list like this

Reply:
Removed iOS case

Reply:
@buyaa-n why is tvOS missing it that list?

Reply:
The intent of the default list is to model what we think most library authors will care about. We didn't add `Browser` because we believe most libraries don't need to run on WebAssembly nor that they can, due to the restrictions of the sandbox.

Whether or not tvOS or iOS are in that group is for the Xamarin team to decide. But the litmus test will be how much code it would flag. If a large chunk of fairly widely used APIs won't work on iOS or tvOS, we probably shouldn't add either to the default set.